Userland/Games/Hearts/SettingsDialog.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Types.h>
+#include <LibGUI/Dialog.h>
+
+class SettingsDialog: public GUI::Dialog {
```

Comment:
```suggestion
class SettingsDialog : public GUI::Dialog {
```

Reply:
Clang-format is okay without the space? O_o

Reply:
Oh? It wasn't earlier. See the GitHub actions log. :)

Reply:
Yeah, clang-format wasn't happy
I just saw your suggestion and thought you posted it because clang-format didn't catch it, my bad.